### ISSUE TITLE
New owner for this library

### ISSUE THREAD

**Issue by catharsisjelly** [BenMorel/ebay-sdk-php#11]
I wrote to @BenMorel asking to turn on Issues so that we as a community of people who are using/contributing to this library try to see how we can keep it running, up-to-date and relevant.

I personally, still rate this as the best library out there for PHP and eBay and I have quite a few suggestions on improvements and so on.

So feel free to drop a comment, or emote this issue if you want to be involved. The idea would be for @BenMorel to "abandon" this and the community start a new fork, possibly not under a single persons account and we organise a few people to manage it. I'm happy to put myself forward as part of the committee for the upkeep and continuing development of it.

I'm tagging in anyone who currently has an outstanding PR as a start and anyone who has updated their fork in the last 6 months.

Tagging
* @workzer
* @xtremevision
* @vt-parhomenko
* @tlaemmlein
* @bmaximenko
* @gwack22
* @ming404
* @jlhernandez1388

**Comment by xtremevision** [comment]
I agree 100%. I don't there's a better api for ebay for php, and this should be maintained. In addition, many people that have invested a lot of code using this api are going to be reluctant to switch to a new api that will require a lot of refactoring. I am one of those, having a project in production using this api and refactoring for a new api would be a huge task, and costly.

I would be happy to assist in maintaining a new "community" fork/repo.

**Comment by BenMorel** [comment]
As I said to @catharsisjelly earlier, I'm very much open to abandoning this library and pointing it to another fork, as I'm no longer using it and don't have much bandwidth or incentive to maintain it.

Because you're the only two people who have come forward, I invite you to coordinate your efforts in creating a new repository and agreeing on how it will be managed. If you want to count me among the maintainers, I'm happy to be there, but I'm also fine with not being part of it. In any case, I probably won't be very involved these days.

Once the new repo is in place and you agree on the terms, I will mark the package as abandoned on both GitHub and Packagist, and redirect to the new fork on both Packagist and in the README.

Hint: the `ebay-sdk-php` organization name seems to be available on GitHub!

**Comment by catharsisjelly** [comment]
Thanks for the input @BenMorel & @xtremevision I'm more than happy to set up a little organisation and look for anyone to partake ion this. I'm a little concerned about the lack of activity from other potential contributors on this thread as like @BenMorel I don't really have that much time to dedicate to it. Hopefully with a community this library can continue to be supported and progress.

**Comment by catharsisjelly** [comment]
@BenMorel it's made - https://github.com/ebay-sdk-php/ebay-sdk-php
Let me know when you have done the bits you need to and then I we can close this issue and begin to move any PR's over or something like that.

**Comment by xtremevision** [comment]
Well done Chris. Mike.
  

  

  

  
  
  
  
  
>

**Comment by BenMorel** [comment]
@catharsisjelly Did you remove the repo? It's gone now.

**Comment by gaiterjones** [comment]
Am happy to help out here if any assistance is required.

**Comment by catharsisjelly** [comment]
@BenMorel I didn't no. Github flagged the organisation and I am not sure why but have my suspicions (because we called it eBay).  I have filed a support ticket about it and will update as soon as I hear something. Sorry!

**Comment by catharsisjelly** [comment]
@BenMorel just an update, still no response from Github despite me trying to bump the ticket. I'm going to try an experiment and setup another organisation (maybe not today) that does not have eBay in the title then see if they flag that one too

**Comment by BenMorel** [comment]
@catharsisjelly Yes, it’s very likely that they prevent this to avoid giving the impression that it’s endorsed by eBay.

Let's see how it goes with another name!

**Comment by catharsisjelly** [comment]
Life has been incredibly busy but I made this organisation a few weeks ago and cloned the repo. So far, no flags on it. - https://github.com/auction-php/ebay-sdk-php
